docs/compendium/substitute.md

```diff
@@ -30,6 +31,7 @@ input of request-body for substitution
 - `substfiles` - array of file substitutions specified by `original` and `xchange`
   - `original` - filename of the file from the base ERC
   - `xchange` - filename of the overlay ERC that will be exchanged for the original file
```

Comment:
i will change that to "base" and "overlay", which is more understandable than "original" and "xchange"